index.bs

```diff
@@ -691,6 +699,9 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
     <li><dfn>Zoom</dfn> is a numeric camera setting that controls the focal length of the lens.  The setting usually represents a ratio, e.g. 4 is a zoom ratio of 4:1.  The minimum value is usually 1, to represent a 1:1 ratio (i.e. no zoom).</li>
 
     <li><dfn>Fill light mode</dfn> describes the flash setting of the capture device (e.g. |auto|, |off|, |on|). <dfn>Torch</dfn> describes the setting of the source's fill light as  continuously connected, staying on as long as {{track}} is active.</li>
+
+    <li><dfn>Image format</dfn> is the ASCII-encoded string in lower case describing the format of an image. The <a>image format</a> string MUST be a [=valid mime type=], whose [=type=] MUST be an [=image media type=] and [=subtype=] MAY be one of the [=IANA registered image subtypes=].
```

Comment:
I think here we should specify that the UA should be able to display the format, similar to what we specify in e.g. [`MediaRecorder`](https://w3c.github.io/mediacapture-record/MediaRecorder.html#dom-mediarecorder-mimetype) .